generated/niscope/niscope.proto

```diff
@@ -979,10 +966,7 @@ message CheckAttributeViInt64Request {
   nidevice_grpc.Session vi = 1;
   string channel_list = 2;
   NiScopeAttribute attribute_id = 3;
-  oneof value_enum {
-    int64 value_raw = 4;
-    NiScopeInt64AttributeValues value = 5;
-  }
+  int64 value_raw = 4;
```

Comment:
Note: For this change, Harsha saw it in an earlier version of importing ni-scope [here](https://github.com/ni/grpc-device/pull/742#discussion_r980734958) and thought this was okay:

"Looking at the P2P_NOTIFY_MESSAGE_PUSH_ADDR attribute documentation, this specifies to which peer address a message should be pushed. It is unlikely someone would have used an enum for this & most likely they would have set a raw value. So, although it is a breaking change the impact may not be big, but rather beneficial? If someone was using the enum by mistake (which was incorrect), they will now be forced to use the raw value. Which seems the right thing to do.
(please correct me if i'm missing something)"